docs/sources/mimir/references/http-api/index.md

```diff
@@ -44,6 +44,7 @@ This document groups API endpoints by service. Note that the API endpoints are e
 | [Tenants stats](#tenants-stats)                                                       | Distributor                    | `GET /distributor/all_user_stats`                                         |
 | [HA tracker status](#ha-tracker-status)                                               | Distributor                    | `GET /distributor/ha_tracker`                                             |
 | [Flush chunks / blocks](#flush-chunks--blocks)                                        | Ingester                       | `GET,POST /ingester/flush`                                                |
+| [Prepare for Shutdown](#prepare-for-shutdown)                                         | Ingester                       | `GET,POST /ingester/prepare-shutdown`                                     |
```

Comment:
This `prepare-shutdown` vs `prepare-for-shutdown` difference bugs me. Can we just call the endpoint `/ingestter/prepare-for-shutdown`? I don't think  there's need to save those extra 5 chars.
This also applies to the code (functions and interfaces).

Reply:
I'm not going to change this.